### ISSUE TITLE
ValueError: Dataframe must have columns 'ds' and 'y' with the dates and values respectively.

### ISSUE THREAD

**Issue by githubhnw** [facebook/prophet#603]
I renamed my dataframe as df and columns ds for timestamp and y for numeric columns and it gives this error:

![image](https://user-images.githubusercontent.com/24211109/42511755-6d6a4f64-845b-11e8-8655-29ef75bfda58.png)

The column types are:

ds: datetime64[ns]
y: float64

**Comment by luofanghao** [comment]
are you sure you make "ds" and "y" in lowercase?

**Comment by githubhnw** [comment]
pretty sure yes

**Comment by githubhnw** [comment]
I restarted the kernel and it seems to work now. The only issue is that it takes a long time so I have to sample my data for fitting because I have a set of 6.5 million records. 

has anyone found a way to fit on big data or to use Prophet with pyspark? I'm just wondering how Facebook uses it on its big data?

Thanks!

**Comment by githubhnw** [comment]
Thanks @bletham , I took 1 day for fitting and next day for forecasting out of almost 2 years of data, with sec frequency in general. My data is not equally spaced, it's made of sensor data. I managed also to reduce the range/interval of forecasted points by setting n_changepoints to 0, because my data is like a stationary timeseries in general, it doesn't have a specific trend over long time, just on some periods.
